ui/src/domain/credential.ts

```diff
@@ -19,7 +19,7 @@ export interface Credential {
 
 export interface IssuedQRCode {
   qrCode: unknown;
-  schemaType?: string;
+  schemaType: string;
```

Comment:
Since this is no longer optional, we can remove the guard on line 68:

```
        {schemaType && (
          <Row>
            <Col
              style={{
                padding: 24,
                paddingBottom: 8,
              }}
            >
              <Typography.Title ellipsis={{ tooltip: true }} level={3}>
                {schemaType}
              </Typography.Title>
            </Col>
          </Row>
        )}
```